zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -172,8 +175,16 @@ public static boolean publish(Collection collection, String email, CollectionRea
                         return false;
                     }
 
+                    Future<Boolean> imageFuture = publishImages(collection);
+
                     publishComplete = publishFilesToWebsite(collection, email, collectionReader);
 
+                    try {
+                        publishComplete &= imageFuture.get().booleanValue();
```

Comment:
Has it been confirmed that failing to publish images will result in the publish being deemed a failure? If I am reading the code correctly, updated pages could be published to the website, but if the images fail then the publish will be labelled as failed, even though the pages have published.